esp/platform/espp.hpp

```diff
@@ -20,13 +20,21 @@
 
 #include "espthread.hpp"
 #include "espcfg.ipp"
+#include "jlog.ipp"
 
 typedef ISocket * isockp;
 
 
 typedef MapBetween<int, int, isockp, isockp> SocketPortMap;
 typedef CopyReferenceArrayOf<ISocket> SocketPortArray;
 
+class CEspElevatedLoggingFilter : public CategoryLogMsgFilter
+{
+public:
+    CEspElevatedLoggingFilter() : CategoryLogMsgFilter(MSGAUD_all, MSGCLS_elevated, DebugMsgThreshold, false) {}
```

Comment:
@kenrowland discussed @timothyklemm 's comment on line 35, and discovered we were actually discussing the DebugMsgThreshold passed in to the constructor. Ultimately I don't believe the detail is considered in the mayIncludeCategory, but for optics we should use a max detail threshold - I'd prefer a constructor with only class but not sure if that's possible.